build-tools/build.sh

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+#!/bin/sh
+
+buildToolsFolder="$(basename "$(dirname "$0")")"
+generatorFolder=$buildToolsFolder/../index/generator
+
+display_usage() { 
+  echo "A devfile registry repository folder must be passed in as an argument" 
+  echo "usage: ./build.sh <path-to-registry-repository-folder>" 
```

Comment:
IMO, probably we can remove the part of path specify: `./` as the script can be run anywhere, so the usage would be `echo "usage: build.sh <path-to-registry-repository-folder>" `

Reply:
👍

Reply:
Updated.